Application/EdFi.Ods.Features/Profiles/ProfileMetadataCacheExpiredNotificationHandler.cs

```diff
@@ -37,5 +74,13 @@ public async Task Handle(ProfileMetadataCacheExpired notification, CancellationT
 
         await _apiJobScheduler.AddSingleExecutionJob<AdminProfileNamesPublisherJob>(
             nameof(ProfileMetadataCacheExpiredNotificationHandler));
+
+        _profilesAwareContractResolver.Clear();
+        _openApiMetadataCacheProvider.ResetCacheInitialization();
+        _mappingContractProvider.Clear();
+
+        // Clears te ApiClientDetails cache, to handle scenarios where a Profile is removed
+        // from the database
+        (_apiClientDetailsInterceptor as IClearable).Clear();
```

Comment:
IClearable should also be implemented in [CachingInterceptor.cs](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/f8257c1292bbb68a8f3cb2a10e4d1d9d623093a4/Application/EdFi.Ods.Common/Caching/CachingInterceptor.cs#L12)
Check [main](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/2e98f64249ffe6406f4a21bd378aaf007d6e8178/Application/EdFi.Ods.Common/Caching/CachingInterceptor.cs#L12)